### PR TITLE
Change copyright comment in java files from /** */ style to /* */.

### PR DESCRIPTION
**Pull request by sjoerdtalsma**
This avoids automatically adding `<P>` between the paragraphs every time an IDE (like IntelliJ) auto-formats the code.

### CODE REVIEW

File: opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java [hunk 1]
```diff
@@ -16,11 +16,7 @@
 import io.opentracing.Span;
 import io.opentracing.SpanContext;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (yurishkuro) [inline opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java]: this is not recommended by many style checks

Comment (sjoerdtalsma) [inline opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java]: Good catch, must've been a side-effect from my IntelliJ settings, I'll correct this!

File: opentracing-util/src/main/java/io/opentracing/util/ThreadLocalActiveSpan.java [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
 import java.util.concurrent.atomic.AtomicInteger;
 
- /**
-  * {@link ThreadLocalActiveSpan} is a simple {@link ActiveSpan} implementation that relies on Java's
-  * thread-local storage primitive.
-  *
-  * @see ActiveSpanSource
-  * @see Tracer#activeSpan()
-  */
+/**
```
Comment (pavolloffay) [inline opentracing-util/src/main/java/io/opentracing/util/ThreadLocalActiveSpan.java]: Did you change this manually?

Comment (sjoerdtalsma) [inline opentracing-util/src/main/java/io/opentracing/util/ThreadLocalActiveSpan.java]: Nope, must have been an IntelliJ autoformat.
By the way, I agree with you on the license-only commit; that was my intention, the formatting 'came free' by the tooling :wink:

**Comment by sjoerdtalsma** [comment]
I'll change the code to "license-only" and try to revert all formatting changes, please don't merge yet.

**Comment by sjoerdtalsma** [comment]
Looks like the accidental auto-import changes were finally reverted.
Sorry guys: turns out I had "Optimize imports" flag on at "before commit" and since all Java files were touched by the license plugin this problem got re-introduced several times even after I explicitly reverted the import changes :man_facepalming:

**Pull request merged**